### PR TITLE
Added find the duplicates from an array

### PR DESCRIPTION
**Pull request by RaiyanMahin**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

YES.

### Description
Given an array of N elements , find all duplicate elements from it. In Competitive Programming, some times we need to do this to solve our problem.

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests
#3283

### CODE REVIEW

**Comment by RaiyanMahin** [comment]
@iamrajiv @ankitaggarwal23 kindly review

File: C-Plus-Plus/cp/find_duplicate_elements_from_an_array.cpp
Comment (ankitaggarwal23) [inline C-Plus-Plus/cp/find_duplicate_elements_from_an_array.cpp]: - Add a few more description. 
- Use Multiline comments

File: C-Plus-Plus/cp/find_duplicate_elements_from_an_array.cpp
Comment (ankitaggarwal23) [inline C-Plus-Plus/cp/find_duplicate_elements_from_an_array.cpp:35]: Add a new line above the int main()

File: C-Plus-Plus/cp/find_duplicate_elements_from_an_array.cpp
Comment (ankitaggarwal23) [inline C-Plus-Plus/cp/find_duplicate_elements_from_an_array.cpp]: Align your code properly

**Review by ankitaggarwal23 — CHANGES_REQUESTED** [review]

**Comment by RaiyanMahin** [comment]
I made all the changes.

**Comment by RaiyanMahin** [comment]
this time 4 checks fail :(
don't know why

**Review by ankitaggarwal23 — APPROVED** [review]

**Comment by ankitaggarwal23** [comment]
@RaiyanMahin You can ignore it, there is some glitch at deepsource's end.

**Comment by ankitaggarwal23** [comment]
> ### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?
> YES.
> 
> ### Description
> Given an array of N elements , find all duplicate elements from it. In Competitive Programming, some times we need to do this to solve our problem.
> 
> ### Checklist
> * [x]  I've read the contribution guidelines.
> * [x]  I've checked the issue list before deciding what to submit.
> * [x]  I've edited the `README.md` and link to my code.
> 
> ## Related Issues or Pull Requests
> #3283

@RaiyanMahin  kindly add your issue no. not the PR no. because after merging we have to close that respective issue

**Review by ankitaggarwal23 — APPROVED** [review]

**Comment by RaiyanMahin** [comment]
@ankitaggarwal23  I didn't made any issue.  I directly made the PR.

**Comment by ankitaggarwal23** [comment]
Means No one assigned you the issue and you have himself assigned and made a PR

**Comment by ankitaggarwal23** [comment]
@RaiyanMahin You can't make direct PR without having an issue assigned to it. This is against the guidelines and unfair with other participants. So I can't merge this without having an issue liked with it. It comes under invalid PR. So I'm closing this now.

**Pull request closed without merging**
